api-reference/10 UI Components/dxSelectBox/1 Configuration/displayExpr.md

```diff
@@ -0,0 +1,7 @@
+#include common-demobutton with {
+    url: "https://js.devexpress.com/Demos/WidgetsGallery/Demo/{WidgetName}/GroupedItems/"
```

Comment:
Are you going to export the link with {WidgetName} to some other description?

Reply:
The TagBox inherits the SelectBox.